doc/api/esm.md

```diff
@@ -1517,6 +1523,18 @@ loaded from disk but before Node.js executes it; and so on for any `.coffee`,
 `.litcoffee` or `.coffee.md` files referenced via `import` statements of any
 loaded file.
 
+You could also chain this loader together with the HTTPS loader from above:
```

Comment:
Have you tested that this works? You could change your example to:

```coffee
# remote.coffee
import { VERSION } from 'https://coffeescript.org/browser-compiler-modern/coffeescript.js'

console.log VERSION
```

And that code should be runnable.

Reply:
well the idea is that it loads coffeescript over https.

Reply:
just tested, it does indeed work.

![](https://gc.gy/59678007.png)

Reply:
Thanks for testing this, that's what I was looking for.